lib/_stream_writable.js

```diff
@@ -148,7 +150,12 @@ function WritableState(options, stream, isDuplex) {
 
   // Storage for data passed to the afterWrite() callback in case of
   // synchronous _write() completion.
-  this.afterWriteTickInfo = null;
+  this[kAfterWriteTickInfo] = null;
+  ObjectDefineProperty(this, 'afterWriteTickInfo', {
```

Comment:
This should go on the prototype and be moved to the parent scope so that it only runs once.

Reply:
Done.